README.md

```diff
@@ -1,2 +1,31 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+```
+var random_words = random words number of which is 8
+*random_words have similar beginnings or endings*
+var random_characters = randomly generated any characters
+var guesses = 0
+
+Display random_words and random_characters on screen 
+Give user ability to click on random_words and random_characters
+chosen_word = system randomly chooses one word from random_words
+
+while guesses < 4
+   if user doesn't click on chosen_word
+      guesses = guesses + 1
+      display how many characters have the same position as the chosen_word
+      user has another chance of choosing a word or character
+   else
+      display another screen saying something funny or clever
+      after clicking any button bring user to main menu where he can try once more
```

Comment:
Does this exit the while loop and end the game? If so, should make this explicit